### PR TITLE
MWPW-128620 FaaS Form 115 Styles

### PR DESCRIPTION
**Pull request by Brandon32**
* Adding styles for FaaS Form 115, cleanup will come later.

Resolves: [MWPW-128620](https://jira.corp.adobe.com/browse/MWPW-128620)

**Test URLs:**
- Before: https://faas-styles--bacom--adobecom.hlx.page/drafts/bmarshal/faas/columns-block/faas-115?martech=off
- After: https://bmarshal-faas-styles--bacom--adobecom.hlx.page/drafts/bmarshal/faas/columns-block/faas-115?martech=off

### CODE REVIEW

**Comment by Brandon32** [comment]
Section columns two-up:
- https://bmarshal-faas-styles--bacom--adobecom.hlx.page/drafts/bmarshal/faas/section-columns/faas-21
- https://bmarshal-faas-styles--bacom--adobecom.hlx.page/drafts/bmarshal/faas/section-columns/faas-21-clearbit
- https://bmarshal-faas-styles--bacom--adobecom.hlx.page/drafts/bmarshal/faas/section-columns/faas-63
- https://bmarshal-faas-styles--bacom--adobecom.hlx.page/drafts/bmarshal/faas/section-columns/faas-79
- https://bmarshal-faas-styles--bacom--adobecom.hlx.page/drafts/bmarshal/faas/section-columns/faas-79-clearbit
- https://bmarshal-faas-styles--bacom--adobecom.hlx.page/drafts/bmarshal/faas/section-columns/faas-115

Columns block:
- https://bmarshal-faas-styles--bacom--adobecom.hlx.page/drafts/bmarshal/faas/columns-block/faas-21
- https://bmarshal-faas-styles--bacom--adobecom.hlx.page/drafts/bmarshal/faas/columns-block/faas-21-clearbit
- https://bmarshal-faas-styles--bacom--adobecom.hlx.page/drafts/bmarshal/faas/columns-block/faas-63
- https://bmarshal-faas-styles--bacom--adobecom.hlx.page/drafts/bmarshal/faas/columns-block/faas-79
- https://bmarshal-faas-styles--bacom--adobecom.hlx.page/drafts/bmarshal/faas/columns-block/faas-79-clearbit
- https://bmarshal-faas-styles--bacom--adobecom.hlx.page/drafts/bmarshal/faas/columns-block/faas-115

File: styles/faas.css [hunk 1]
```diff
@@ -17,8 +17,45 @@
...
+  }
+}
+
+/* desktop up */
+@media screen and (min-width: 1200px) {
+  .resource-form.section.two-up,
+  .resource-form .columns > .row {
+    grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));
+  }
+}
```
Comment (JasonHowellSlavin) [inline styles/faas.css:49]: please move these styles to `media query min-width 1200px` at line 384

**Review by JasonHowellSlavin — APPROVED** [review]
Looks great, few comments.

There may be more areas where we can use vars, but I am not sure about them TBH, I feel like I heard talk about changing the var system. Also, for designs not consonant based, using consonant designed vars feels...odd?

File: styles/faas.css [hunk 2]
```diff
@@ -277,11 +349,28 @@
     grid-template-columns: 1fr 1fr 1fr;
   }
 
+  .faas.faas-form-115 label,
+  .faas.faas-form-115 input:not([type=checkbox]):not([type=radio]):not([type=submit]),
```
[the remark this answers is not in the document]
Comment (Brandon32) [reply styles/faas.css:353]: This is a FaaS specificity issue, I would like to refactor to make the specificity more readable by including something like `.resource-form .faas.faas-form-115 .faas-form input`, but was running into issues with the specificity other specific form id styles.

[the remark this answers is not in the document]
Comment (JasonHowellSlavin) [reply styles/faas.css:353]: Forms. Yay!

File: styles/faas.css [hunk 3]
```diff
@@ -261,7 +321,13 @@
   margin-right: calc(var(--grid-margins-width) * -1 / 2);
 }
 
+/* tablet up */
```
Comment (meganthecoder) [inline styles/faas.css:324]: Adding comments for breakpoints seems superfluous

**Review by meganthecoder — APPROVED** [review]
Just a couple notes about the comments

File: styles/faas.css [hunk 4]
```diff
@@ -17,8 +17,34 @@
...
+  line-height: 28px;
+  border-radius: 5px;
+}
+
+.resource-form .faas {
+  margin-left: calc(var(--grid-margins-width) * -1 / 2);
+  margin-right: calc(var(--grid-margins-width) * -1 / 2);
+}
+
 /** next styles above due to specificity, form specific styles below **/
```
[the remark this answers is not in the document]
Comment (Brandon32) [reply styles/faas.css:40]: I'll remove the comments

File: styles/faas.css (hunk 3 above)
Comment (Brandon32) [reply styles/faas.css:324]: I'll remove the comments

**Pull request merged**